### PR TITLE
Nginx config update for caching issue

### PR DESCRIPTION
**Pull request by haneef-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: deriv.com.conf [hunk 1]
```diff
@@ -1,13 +1,16 @@
 server {
     listen 80;
     server_name  _;
+    #### Default Cache headers. This to prevent caching non-existent pages while releasing new versions. Reference -  https://www.gatsbyjs.com/docs/caching/  #######
+    add_header Cache-Control "public, max-age=0, no-cache, no-store, must-revalidate" always;
+    add_header X-CF-IPCOUNTRY $http_cf_ipcountry;
```
Comment (afshinpaydar-binary) [inline deriv.com.conf:6]: Why we should return `X-CF-IPCOUNTRY` header? this should add by Cloudflare

Comment (gareth-deriv) [reply deriv.com.conf:6]: Nice catch. I think we were snow blind.

File: deriv.com.conf [hunk 2]
```diff
@@ -1,13 +1,16 @@
 server {
     listen 80;
     server_name  _;
+    #### Default Cache headers. This to prevent caching non-existent pages while releasing new versions. Reference -  https://www.gatsbyjs.com/docs/caching/  #######
```
Comment (gareth-deriv) [inline deriv.com.conf:4]: Comment style is unusual to me.
```suggestion
    # By default not cache for most resources, recommended by Gatsby: https://www.gatsbyjs.com/docs/caching/ 
   # As well as being recommended by Gatsby, this ensures our releases take immediate effect
```

File: deriv.com.conf [hunk 3]
```diff
@@ -30,21 +33,10 @@ server {
...
     
-    location ~ (app-data\.json)$ {
-        add_header Cache-Control "public, max-age=0, must-revalidate";
-    }
-
-    location /page-data {
-        add_header Cache-Control "public, max-age=0, must-revalidate";
-    }
-
+    #### Cache the content for 365 days as these are static content #####
```
Comment (gareth-deriv) [inline deriv.com.conf:39]:
```suggestion
    # Cache the truly static content for 365 days
```

**Review by gareth-deriv — CHANGES_REQUESTED** [review]
Good work.

Requested Change is removing the CF country.

Niggles are the comments. I've made suggestions. It's up to you if you want to implement them or not 😄

**Pull request closed without merging**